client/src/views/UploadTemplate.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Form, Button, Modal } from 'react-bootstrap';
+import axios from 'axios';
+
+import './UploadTemplate.css';
+import { Input } from '@material-ui/core';
+
+class UploadTemplateModal extends React.Component {  
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      curFileName: null
```

Comment:
could you rename this to `currentFileName`

Reply:
Done